src/main/java/io/cryostat/net/web/http/api/v2/CertificatePostHandler.java

```diff
@@ -151,10 +151,11 @@ public boolean isOrdered() {
             throw new ApiException(409, filePath.toString() + " Certificate already exists");
         }
 
-        try (InputStream fis = fs.newInputStream(certPath);
-                FileOutputStream out = outputStreamFunction.apply(filePath.toFile())) {
-
+        try (InputStream fis = fs.newInputStream(certPath)) {
             Collection<? extends Certificate> certificates = certValidator.parseCertificates(fis);
+
+            FileOutputStream out = outputStreamFunction.apply(filePath.toFile());
```

Comment:
We still need to be sure we call `.close()` on `out` when we're done with it, so it's probably best if it is also inside of a `try-with-resources`.